lib/services/oauth/server.go

```diff
@@ -68,6 +70,11 @@ func SetUpOauth() {
 	clientStore = store
 }
 
+func ValidateToken(request *http.Request) (tokenInfo oauth2.TokenInfo, err error) {
+	tokenInfo, err = oauthServer.ValidationBearerToken(request)
+	return
+}
```

Comment:
Can you follow up with this one?
https://nimblehq.co/compass/development/code-conventions/golang/#functions

Reply:
Thanks fixed in [18306bc](https://github.com/carryall/go-google-scraper-challenge/pull/93/commits/18306bc382b75ca268ad66c085b77ab796a365c7)